app/components/ui/Banner/Banner.scss

```diff
@@ -2,16 +2,25 @@
 
 .bannerContainer {
   width: 100%;
-  padding: calc-em(20px) calc-em(30px);
+  padding: calc-em(16px) calc-em(28px);
   color: black;
   background-color: $color-yellow;
   font-size: calc-em(24px);
   text-align: center;
+  position: sticky;
+  span {
+    display: inline-block;
+    vertical-align: middle;
+    line-height: normal;
+  }
   strong {
     font-weight: bold;
   }
-  @media screen and (max-width: 800px) {
-    font-size: 16px;
+  @media screen and (max-width: 600px) {
+    font-size: calc-em(16px);
+    height: 120px;
+    line-height: 120px;
+    padding: 0 calc-em(28px);
```

Comment:
I'm not sure that the styling on the span or anything in this media query (other than the font-size) is actually necessary. I wouldn't add it if we don't need it.

Reply:
Oh i see now, you were doing this so that you could specify the banner height. I'm gonna mess with this a bit and see if I can figure out a less brittle way to do this